Source/GameEngine/LudoContext.cs

```diff
@@ -0,0 +1,44 @@
+﻿using GameEngine;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace TheLudoGame
+{
+    class LudoContext: DbContext
+    {
+        private IConfigurationRoot _configuration;
+        
+        public DbSet<Board> Board { get; set; }
+        public DbSet<Game> Game { get; set; }
+        public DbSet<Player> Player { get; set; }
+        public DbSet<Piece> Piece { get; set; }
+
+        public LudoContext() { }
+
+        public LudoContext(IConfigurationRoot configuration)
+        {
+            this._configuration = configuration;
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+            optionsBuilder.EnableSensitiveDataLogging(true);
```

Comment:
Why have you decided to enable this?

Reply:
Borttaget nu, följde något SO exempel som föreslog det för att se tydligare felutskrift ifall jag inte minns fel